tools/CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+### Contributing OpenAPI generation tools
+
+## Changes in tools
+
+We need to regenerate our SDK every time tools are changed as - changes might affect the generated clients.
+PR based checks will verify if changes are up to date.
+
+```
+make clean_and_generate
+```
+
+### OpenAPI generator
+
+OpenAPI generator version is set in [./openapitools.json](./openapitools.json) file.
+
+### Transformer
+
+Follow [transformer readme](./transformer/README.md) for more information
+
+#### Modification of the custom templates
```

Comment:
seem comment about levels on tittles, this doesn't seem related to `Transformer` so the level should be ajusted